src/Files.App/Data/Models/ItemViewModel.cs

```diff
@@ -76,6 +76,13 @@ public sealed class ItemViewModel : ObservableObject, IDisposable
 			private set => SetProperty(ref currentFolder, value);
 		}
 
+		private string? solutionFilePath;
+		public string? SolutionFilePath
+		{
+			get => solutionFilePath;
+			private set => SetProperty(ref solutionFilePath, value);
+		}
```

Comment:
Proeprty-backing-field name must start with underscore and be UpperCase.

```suggestion
		private string? _SolutionFilePath;
		public string? SolutionFilePath
		{
			get => _SolutionFilePath;
			private set => SetProperty(ref _SolutionFilePath, value);
		}
```